pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
   flutter:
     sdk: flutter
   latlong2: ^0.8.0
+  flutter_map_marker_popup: ^0.3.0
```

Comment:
@rorystephenson
`flutter_map_marker_popup` can be updated to ^1.0.0 now as it is null-safe.
`flutter_map` can be updated to ^0.3.1

Thanks for all the fixes! Looking forward for the null-safety release of this plugin.

Reply:
Updated!